.github/workflow/nuget.yml

```diff
@@ -5,16 +5,23 @@ on:
       - master
 jobs:
   build:
-    runs-on: ubuntu-latest
     name: Build, Pack & Publish
+    runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout
+        uses: actions/checkout@v2
       - name: Setup dotnet
-        uses: actions/setup-dotnet@v1
+        uses: actions/setup-dotnet@v1.5.0
         with:
           dotnet-version: '5.0.x'
-      - run: dotnet build csharp-htk
-      - name: Create the package
-        run: dotnet pack --configuration Release csharp-htk
-      - name: Publish the package to GPR
-        run: dotnet nuget push csharp-htk/bin/Release/*.nupkg -k ${{secrets.NUGET_AUTH_TOKEN}} -s https://api.nuget.org/v3/index.json
\ No newline at end of file
+      - name: Install dependencies
+        run: dotnet restore
+        working-directory: ./
+      - name: Build
+        run: dotnet build --configuration Release --no-restore
+        working-directory: ./
+      - name: Publish
+        uses: brandedoutcast/publish-nuget@v2.5.2
```

Comment:
Why are we using this one?

https://github.com/brandedoutcast/publish-nuget seems to point to: https://github.com/marketplace/actions/publish-nuget / `rohith/publish-nuget`

Reply:
rohith is a placeholder account that redirects to brandedoutcast. So They are same.